### ISSUE TITLE
Bundle fails with webpack 5.1.3: Module not found: Error: Can't resolve 'target ...'

### ISSUE THREAD

**Issue by Kah0ona** [bhauman/figwheel-main#266]
With webpack 5.1.3 and bundling configured (via :target :bundle) I get (both on macOS and Linux) this error:

````
[Figwheel] Successfully compiled build dev to "target/public/cljs-out/dev/main.js" in 28.101 seconds.
[Figwheel] Bundling: npx webpack --mode=development target/public/cljs-out/dev/main.js -o target/public/cljs-out/dev/main_bundle.js
[Figwheel:SEVERE] Bundling command failed
asset bundle.js 644 bytes [emitted] (name: main)
ERROR in main
Module not found: Error: Can't resolve 'target/public/cljs-out/dev/main.js' in '/Users/marten/Sites/clojure/aviationglass'
webpack 5.1.3 compiled with 1 error in 60 ms
````

It turns out the npx command needs a `./` prefix at `target/public/cljs-out/dev/main.js`

So if you use the default setting (ie. no :output-dir configuration in `dev.cljs.edn`) this will fail. 

**The workaround is** to add this to your `dev.cljs.edn`:
````
{:output-to  "./target/public/cljs-out/dev/main.js"
  ...}
````

**Comment by bhauman** [comment]
This is fixed in 0.2.12-SNAPSHOT

Heads up though, when I updated to webpack-cli to latest. It failed to parse the standard webpack CLI command correctly.  Don't know whats going on there, but keep it in mind.

**Comment by bhauman** [comment]
They changed the meaning of the "-o" flag to "--output-path" and got rid of the shorthand version that allows you to pass the path and the filename as one thing.

This fix is deployed to 0.2.12-SNAPSHOT

**Comment by wcalderipe** [comment]
I've just tested the ` 0.2.12-SNAPSHOT` with the following `dev.cljs.edn` and it's working like a charm back again.

```edn
^{:auto-bundle      :webpack
  :watch-dirs       ["test" "src"]
  :css-dirs         ["resources/public/css"]
  :auto-testing     true
  :extra-main-files {:cards {:main lab-figwheel.cards}}}
{:main lab-figwheel.core
 :devcards true}
```
